### ISSUE TITLE
Support v5.8.0

### ISSUE THREAD

**Issue by craftzdog** [TaylanTatli/inkdrop-bera-light-ui-theme#1]
Hi TaylanTatli!

I just released v5.8.0-beta.1, which comes with a bunch of UI improvements:

https://forum.inkdrop.app/t/inkdrop-desktop-v5-8-0-beta-1/4473

It has some accessibility improvements, which require custom themes to update.
The release notes mention how to update your theme.
Could you check it out?

Thanks for creating a great theme!

**Comment by TaylanTatli** [comment]
Sure. Thanks for your hard works. @TolgaTatli can you help me for this?

**Comment by TolgaTatli** [comment]
> Sure. Thanks for your hard works. @TolgaTatli can you help me for this?

Absolutely. I can help you to update your theme.

**Comment by TaylanTatli** [comment]
@craftzdog I can't publish the new version because it asks for API Token, but there is no such thing on my inkdrop page.

**Comment by craftzdog** [comment]
Are you logged in with Inkdrop on the app? `ipm` uses the credentials stored in the app's keystore.

**Comment by TaylanTatli** [comment]
> Are you logged in with Inkdrop on the app? `ipm` uses the credentials stored in the app's keystore.

I did, but with different mail. As you can see [here](https://forum.inkdrop.app/t/login-problem-invalid-encyption-key/3873), I had created a new account because of that problem.

**Comment by craftzdog** [comment]
makes sense. I've migrated the owner info of your plugins to your new account. Can you try it again?

**Comment by TaylanTatli** [comment]
Done.

**Comment by craftzdog** [comment]
I guess the theme isn't compiled to the `styles` folder.

https://github.com/TaylanTatli/inkdrop-bera-light-ui-theme/tree/master/styles

Have you tested it on the app?

**Comment by TaylanTatli** [comment]
Sorry @craftzdog, I will check again.

**Comment by TaylanTatli** [comment]
Reverted everything. I'll do it all over again.

**Comment by craftzdog** [comment]
You can run `npm run build` before `ipm publish`.

https://developers.inkdrop.app/guides/create-a-theme#creating-a-ui-theme
